### DISCUSSION TITLE
Release Static Web page to Host

### DISCUSSION THREAD

**Discussion by dragonballprogramming** [community/community#40930] [Repositories]
### Select Topic Area

Product Feedback

### Body

I am trying to have github build my site and then push the static pages to Ionos Deploy Now system. I am using Nodejs, Express, and EJS for my stack. There is no DB as its a basic site. Ionos wants a static site pushed to them in the form of html pages. How do I setup the build so that github builds the site and deploys static webpages to the host.

**Comment by ossanna16** [comment]
Hi there @dragonballprogramming and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by facundofarias** [comment]
You can just start with DeployHQ: https://www.deployhq.com/guides/ionos
